stmocli/stmo.py

```diff
@@ -18,6 +18,28 @@ def __init__(self, redash_api_key, conf=None):
         self._redash = RedashClient(redash_api_key)
         self.redash_api_key = redash_api_key
 
+    def start_query(self, file_name, data_source_id=1, description=None):
+        """Add a query created locally to .stmocli.conf.
```

Comment:
A little bikeshedding:
* Should this method also push the new query to STMO?
* `start` sounds a little bit like executing a query to me; maybe `create`?